### PR TITLE
add auto suspend

### PR DESCRIPTION
**Pull request by EternalQ**

### CODE REVIEW

**Comment by slntopp** [comment]
- Add Auto-Resume setting
 - Add Auto-Resume

File: pkg/billing/routines.go [hunk 1]
```diff
@@ -124,11 +209,122 @@ func (s *BillingServiceServer) GenTransactionsRoutine(ctx context.Context) {
...
 
+		cursor, err := s.db.Query(ctx, accToSuspend, map[string]interface{}{
+			"conf": suspConf,
+		})
+		if err != nil {
+			log.Error("Error Quering Accounts to Suspend", zap.Error(err))
+		}
+		defer cursor.Close()
+
+		for {
```
Comment (ijimiji) [inline pkg/billing/routines.go:220]: Why not `for cursor.HasMore() {` instead of catching error?

File: pkg/registry/proto/registry.proto [hunk 2]
```diff
@@ -59,6 +59,16 @@ service AccountsService {
...
+    rpc Suspend(nocloud.registry.accounts.SuspendRequest) returns (nocloud.registry.accounts.SuspendResponse) {
+        option (google.api.http) = {
+            get: "/accounts/{uuid}/suspend"
+        };
+    };
+    rpc Unsuspend(nocloud.registry.accounts.UnsuspendRequest) returns (nocloud.registry.accounts.UnsuspendResponse) {
+        option (google.api.http) = {
+            get: "/accounts/{uuid}/unsuspend"
+        };
+    };
```
Comment (ijimiji) [inline pkg/registry/proto/registry.proto:71]: Why use `get` method even though we are already using post in modules?
https://github.com/slntopp/nocloud/blob/0f3bef2666c09c654f04f79c519f895d2241c417/pkg/services/proto/services.proto#L170-#L192

**Review by ijimiji — APPROVED** [review]
👍

**Pull request merged**